client/src/components/Schedule/ScheduleView/ScheduleView.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { CourseEvent } from 'services/course';
+import { TableView } from '../TableView';
```

Comment:
Perhaps, it's easier to do it like this?
```suggestion
import { TableView as DEFAULT_SCHEDULE_VIEW } from '../TableView';
```

Reply:
I think const a bit more clear.